include/capstone/capstone.h

```diff
@@ -765,7 +786,7 @@ const char *CAPSTONE_API cs_strerror(cs_err code);
  which complicates things. This is especially troublesome for the case @count=0,
  when cs_disasm() runs uncontrollably (until either end of input buffer, or
  when it encounters an invalid instruction).
- 
```

Comment:
Can you please extract these two formatting changes and open a PR with it?

I'd like to merge something from you so the CI triggers automatically.